baseplate/__init__.py

```diff
@@ -3,14 +3,23 @@
 
 from types import TracebackType
 from typing import Any
+from typing import Callable
 from typing import Dict
+from typing import List
 from typing import NamedTuple
 from typing import Optional
 from typing import Tuple
 from typing import Type
+from typing import TYPE_CHECKING
 
 from baseplate.lib import config
-from baseplate.lib import warn_deprecated
+from baseplate.lib import metrics
+
+
+if TYPE_CHECKING:
+    import baseplate.clients
+    import baseplate.observers.tracing
+    import raven
```

Comment:
I'm pretty sure it doesn't, but does this have any impact on the "required-ness" of the raven library? This just means that you will need to have raven installed in order to perform type checking right? Nothing changes for actual users of the library?

Reply:
correct, `TYPE_CHECKING` will always be `False` at runtime so this code is only ever seen by static analysis tooling.